pkg/grpc/atttestation-api.proto

```diff
@@ -5,21 +5,15 @@ option go_package = "github.com/DIMO-Network/attestation-api/pkg/grpc";
 package grpc;
 
 service AttestationService {
-  rpc BatchCreateVINVC(BatchCreateVINVCRequest) returns (BatchCreateVINVCResponse);
```

Comment:
I guess Yev wants to start with non-batch. Casing looks a bit odd but follows [the official style guide](https://protobuf.dev/programming-guides/style/#message-field-names).